documentation/src/docs/pravega-concepts.md

```diff
@@ -376,120 +280,108 @@ The following figure depicts the components deployed by Pravega:
+- **Tier 1:** It provides short term, low-latency, data storage, guaranteeing the durability of data written to streams. Pravega uses [Apache Bookkeeper](http://bookkeeper.apache.org/) to implement
+Tier 1 storage.
+- **Tier 2:** It provides long term storage for stream data. Pravega uses HDFS, Dell EMC's Isilon or Dell EMC's Elastic Cloud Storage (ECS) to implement Tier 2 Storage.
+
+Tier 1 Storage typically runs _within_ the Pravega cluster. Tier 2 Storage is normally deployed _outside_ the Pravega cluster. Tiering storage is important to deliver the combination of fast access to stream
+data and also allows large data storage for streams. Tier 1 storage
+persists on the most recently written stream data. As data in Tier 1 storage ages,
+it is moved into Tier 2 storage.
 
 Pravega uses [Apache Zookeeper](https://zookeeper.apache.org/) as the
 coordination mechanism for the components in the Pravega cluster.  
 
-Pravega is built as a data storage primitive first and foremost.  Pravega is
-carefully designed to take advantage of software defined storage so that the
+Pravega is built as a data storage primitive first and foremost. Pravega is
+carefully designed to take advantage of software defined storage, so that, the
 amount of data stored in Pravega is limited only by the total storage capacity
-of your data center.  And like you would expect from a storage primitive, once
-data is written to Pravega it is durably stored.  Short of a disaster that
-permanently destroys a large portion of a data center, data stored in Pravega is never
+of the data center. Once the data is written to Pravega it is durably stored.  Short of a disaster, that
+permanently destroys a large portion of a data center, but data stored in Pravega is never
 lost.
 
-Pravega provides a client library, written in Java, for building client-side
-applications such as analytics applications using Flink. The Pravega Java Client
-Library manages the interaction between application code and Pravega via a
+Pravega provides a **Java client library**, for building client-side
+applications such as analytics applications using Flink. The Pravega Java client
+library manages the interaction between the application code and Pravega via a
 custom TCP wire protocol.
 
 
 ## Putting the Concepts Together
 
-The concepts in Pravega are summarized in the following figure:
+The concepts in Pravega are depicted in the following figure:
 
 ![State synchroner](img/putting.all.together.new.png) 
 
--   Pravega clients are Writers and Readers.  Writers write Events into a
-    Stream.  Readers read Events from a Stream.  Readers are grouped into
-    ReaderGroups to read from a Stream in parallel.
+-   Pravega clients are **Writers** and **Readers**.  Writers write events into a
+    stream. Readers read events from a stream. Readers are grouped into
+    **Reader Groups** to read from a stream in parallel.
```

Comment:
Please, do not make extensive use of bold font. It is ok the first time a concept is introduced or as a header of an itemized paragraph, but do not mix bold and normal fonts for the same word all around the text.